app/views/pathways/_challenge.html.erb

```diff
@@ -2,6 +2,9 @@
 <% challenge_template = capture do %>
    <div class='m-6 border p-4 flex flex-col hover:border-blue-500 w-full'>
       <h3 class='font-bold'><%= challenge.title %></h3>
+      <p><% challenge.tags.each do |tag| %>
+         (<%= tag.name %>) 
```

Comment:
if you could add a tag class, we can style it in sass later

Reply:
I just mean something like `:class => 'class'`, won't add any styling but will make it easier for those who do later on. Can always add later though.